lib/_007/Parser/Actions.pm

```diff
@@ -409,6 +413,14 @@ class _007::Parser::Actions {
         make $<EXPR>.ast;
     }
 
+    method pair:quoted ($/) {
+        make Q::Property.new($<key>.made, $<value>.made);
+    }
+
+    method pair:sym ($/) {
```

Comment:
How about these?

* `property:str-expr` (for the `"key": value` case)
* `property:ident-expr` (`key: value`)
* `property:ident` (`key`)
* `property:method`

Some inspiration taken from the [ES6 spec](http://www.ecma-international.org/ecma-262/6.0/ECMA-262.pdf), page 144.